### ISSUE TITLE
Implement an Initial Answer Matching Function

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#30]
## User story
1. As a developer
2. I need to develop a tool that can match answers to the extracted or generated questions
3. So that the training data can be Q&A pairs

## Acceptance criteria
* Each Q from the previous user story has at least 1 corresponding A

* Examine the quality and relevance of answers
    * Log the errors
    * Hold the broken texts to avoid disturbing

* Document the matching logic and limitations for further development
    * observed problems
    * extraction strategy
    * matching algorithm or mechanism

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by christianwielenberg** [comment]
can we close this?
@grayJiaaoLi
